application/common/models/User.php

```diff
@@ -292,7 +292,7 @@ private function validatePassword(): Closure
     {
         return function ($attributeName) {
             $currentPassword = $this->currentPassword ?? new Password();
```

Comment:
Where is $this->currentPassword defined? PHP 8.x doesn't allow automatic creation of properties.

Reply:
Yii 2.x relationship defined in UserBase.